### ISSUE TITLE
Widget does not update

### ISSUE THREAD

**Issue by rugk** [dmfs/opentasks#327]
When I add a new task via widget or the main app, the widget does not update and still shows 0 tasks.
Sometimes it flickers a bit after adding that task, but that's all.

**Comment by dmfs** [comment]
Which one of the widgets do you use? The small one or the large one? Have your tried if the same issue exists with the other one?

**Comment by rugk** [comment]
The small one, I'll try it later.

**Comment by dmfs** [comment]
when you created the Widget you had to select a number of lists, do these lists still exist and are they enabled?
Does it show any tasks when you recreate the widget?

**Comment by rugk** [comment]
> Does it show any tasks when you recreate the widget?

Yes, this always works.
And, as I saw, also closing tasks work, so they disappear.

**Comment by patricksettle** [comment]
@rugk are you syncing with caldav and if so what do you use to sync?  And have you ever noticed the task list temporary showing 0 task before populating in the app? 

also...

If you remove the widget, open the app and confirm they task are shown and re-add the widget does it repopulate? 

I'm new to the app and wondering if this is the same issue I'm experiencing.

**Comment by rugk** [comment]
No, I think for now it is also working here. Still leave this issue open in case anyone else has it/can reproduce it.

**Comment by Chidhambararajan** [comment]
Ok I am using a Samsung m20 with android 10, i am too having this issue or similar I guess 

The smaller and the bigger widgets.. both aren't showing any tasks which is visible in the main tasks app
